store/tikv/coprocessor.go

```diff
@@ -528,7 +534,7 @@ func (worker *copIteratorWorker) run(ctx context.Context) {
 			respCh = task.respChan
 		}
 		worker.handleTask(ctx, task, respCh)
-		if worker.respChan != nil {
+		if worker.respChan != nil && worker.actionOnExceed.isEnabled() {
```

Comment:
Why add the check here? I think it will cause deadlock, since it will never putToken() in non-order cases.